apigo/api/preview.go

```diff
@@ -31,8 +31,8 @@ func init() {
 
 // PreviewRequest is the body of the POST /preview request.
 type PreviewRequest struct {
-	ResourceID       string   `json:"resource_id"`
-	PrimaryKeyValues []string `json:"primary_key_values"`
+	ResourceID       string        `json:"resource_id"`
+	PrimaryKeyValues []json.Number `json:"primary_key_values"`
```

Comment:
So this accepts both ["1"] and [1] ? That's cool

Reply:
it does, but it was not enough! I chose json.RawMessage instead (which is equivalent to []byte) so that the keys can be of arbitrary type